### PR TITLE
Improve dashboard documentation (part 1)

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

With #10995 merged, I'm moving ahead with breaking all our dashboard/web server content into a new series docs, starting with the following:

- **How the dashboard works**, which explains the fundamentals of the dashboard and charts.
- **Interact with charts**, which outlines all the different ways to interact with charts.

This information already exists elsewhere, but not completely obvious formats/locations. 

Once this PR is merged, I'll move on to the next batch. The next batch is **Chart dimensions, contexts, and families** and **Select timeframes to visualize**, followed by **Import, export, and print snapshots** and **Customize the standard dashboard**, and lastly an improved **web server reference**. Once those are all merged, I'll remove obsolete docs and change links to the new files.

##### Component Name

area/docs
area/web

### CODE REVIEW

File: docs/dashboard/how-dashboard-works.mdx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+To zoom, press and hold `Shift`, then use your mouse's scroll wheel, or a two-finger pinch if you're using a touchpad.
+
+See [interact with charts](/docs/dashboard/interact-charts.mdx) for all the possible ways to interact with the charts on
+your dashboard.
+
+## Alarms
+
+Many of the preconfigured charts on the Netdata dashboard also come with preconfigured alarms. Netdata has three primary
+alarm states: `CLEAR`, `WARNING`, and `CRITICAL`. If an alarm moves from a `CLEAR` state to either `WARNING` or
```
Comment (thiagoftsm) [inline docs/dashboard/how-dashboard-works.mdx:79]: Hello @joelhans , Netdata has more than 3 alarms states, but we send `CLEAR`, `WARNING` and `CRITICAL`, when users access the endpoint `/api/v1/alarms?all` they will also see `UNINITIALIZED`, when Netdata starts to run. I think the ideal would be to say `Netdata sends three alarm states`

Comment (ilyam8) [reply docs/dashboard/how-dashboard-works.mdx:79]: Those three are primary. All [Alarm statuses](https://learn.netdata.cloud/docs/agent/health/reference#alarm-statuses)./

Comment (joelhans) [reply docs/dashboard/how-dashboard-works.mdx]: Thanks for the feedback, @thiagoftsm! Updated this section.

**Review by thiagoftsm — APPROVED** [review]
After the last commit, I am approving the PR. LGTM!

**Review by OdysLam — APPROVED** [review]

**Pull request merged**